src/lib/index.js

```diff
@@ -350,19 +350,25 @@ lib.noneOrAll = function(containerIn, containerOut, attrList) {
     }
 };
 
-/** merge data array into calcdata items
+lib.mergeArray = function(traceAttr, cd, cdAttr) {
```

Comment:
> That said, replacing all Lib.mergeArray calls with Lib.fillArray should not break anything. I could do that if you prefer. It might slow things down a little bit when e.g.

I'm not worried about perf - yes, shorter attribute array is an edge case I'm comfortable neglecting. But it does change behavior, even if it's behavior we expect users not to be depending on. I'd leave it as is for the moment, for existing uses, and make an issue about it. We'll probably have to look at each attribute individually to see if we think the change is sufficiently unambiguous as an improvement that it's worth the minor incompatibility.